hunting-buddy.lic

```diff
@@ -60,7 +60,7 @@ class HuntingBuddy
       wait_for_script_to_complete('bescort', @exit) if @exit
       release_cyclics
     end
-    walk_to(@settings.safe_room)
+    walk_to((@settings.training_rooms || [@settings.safe_room]).sample)
```

Comment:
This will pick a random room, but then crossing-training will pick another (likely different) random room. Do you think that is a problem?